client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -79,17 +79,21 @@ export const getTask = (
 		case CHECKLIST_KNOWN_TASKS.EMAIL_VERIFIED:
 			taskData = {
 				timing: 1,
-				title: translate( 'Confirm your email address' ),
+				title: translate( 'Confirm your email address {{emailAddress /}}', {
```

Comment:
🤔  What will happen if the translator decides that the place of the `emailAddress` component is in the beginning of the string? That may be true for some languages. Should it be positioned outside of the translatable string?

Reply:
That will lead to the email displaying above the actual title `Confirm your email address` which is not what we want, 

`Should it be positioned outside of the translatable string?`, it should still be part of the title key in the taskData object, but moving it out like `	title: translate( 'Confirm your email address' ) + userEmail,` will not help as we then can not target the email with css, so how/what would you suggest? @tyxla

Reply:
I'm not sure I understand what you mean - by moving it outside of the `translate()` call to the end of the expression it still remains part of the `title` property? So we should be able to style it? Or am I misunderstanding you?

Reply:
in the latest commit I am displaying the email outside of the translation, hope it is acceptable.

Reply:
Yes, I think it looks great now!